### PR TITLE
additions for user creation

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 1]
```diff
@@ -160,6 +162,32 @@ Keep in mind that Pods that need extensive Security Contexts need to run with a
...
+<cn-prefix>.user.api.<cluster-domain>
+```
+
+where `<cn-prefix>` is a username of your choice and `<cluster-domain>` is your cluster's domain, e.g. `rlbz2.k8s.ginger.eu-central-1.aws.gigantic.io`.
+
+When binding roles to a user you need to use the full username mentioned in the above.
+
+### Using Organizations as Groups in Kubernetes
+
+Organizations you set when creating key-pairs or kubeconfigs, get mapped to groups inside Kubernetes.
```
Comment (marians) [inline content/guides/securing-with-rbac-and-psp/index.md]: Is that comma needed?

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 2]
```diff
@@ -160,6 +162,32 @@ Keep in mind that Pods that need extensive Security Contexts need to run with a
...
+
+Setting a Common Name Prefix results in a username like the following:
+
+```nohighlight
+<cn-prefix>.user.api.<cluster-domain>
+```
+
+where `<cn-prefix>` is a username of your choice and `<cluster-domain>` is your cluster's domain, e.g. `rlbz2.k8s.ginger.eu-central-1.aws.gigantic.io`.
+
+When binding roles to a user you need to use the full username mentioned in the above.
```
Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: mentioned in the above -> mentioned above.

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 3]
```diff
@@ -160,6 +162,32 @@ Keep in mind that Pods that need extensive Security Contexts need to run with a
...
+
+### Using Organizations as Groups in Kubernetes
+
+Organizations you set when creating key-pairs or kubeconfigs, get mapped to groups inside Kubernetes.
+
+You can then asign roles to a whole group of users.
```
Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: asign -> assign

File: content/guides/securing-with-rbac-and-psp/index.md (hunk 1 above)
Comment (puja108) [reply content/guides/securing-with-rbac-and-psp/index.md]: not sure, I guess not ;)

**Pull request merged**